lib/util/git/index.ts

```diff
@@ -837,6 +837,13 @@ export async function commitFiles({
       error.validationMessage = `Renovate cannot push to its branch because branch protection has been enabled.`;
       throw error;
     }
+    if (err.message.includes('can only push your own commits')) {
+      const error = new Error(CONFIG_VALIDATION);
+      error.validationSource = branchName;
+      error.validationError = 'Access denied';
+      error.validationMessage = err.message;
```

Comment:
Let's make it more descriptive

Reply:
'Renovate cannot push to its branch because has no write access'?

Reply:
Renovate has experienced the following error when attempting to push its branch to the server: "....."

And then include the error message inside the quotes